gammapy/makers/utils.py

```diff
@@ -82,11 +82,13 @@ def _get_fov_coords(pointing, irf, geom, use_region_center=True, obstime=None):
                 altaz_coord.az, altaz_coord.alt, pointing_altaz.az, pointing_altaz.alt
             )
         elif irf.fov_alignment == FoVAlignment.RADEC:
-            # Create OffsetFrame
-            frame = SkyOffsetFrame(origin=pointing_icrs)
-            pseudo_fov_coord = sky_coord.transform_to(frame)
-            fov_lon = pseudo_fov_coord.lon
-            fov_lat = pseudo_fov_coord.lat
+            fov_lon, fov_lat = sky_to_fov(
+                sky_coord.icrs.ra,
+                sky_coord.icrs.dec,
+                pointing_icrs.icrs.ra,
+                pointing_icrs.icrs.dec,
+            )
+            fov_lon = -fov_lon
```

Comment:
so what's the correct sign ?

Reply:
GADF specs are here: https://gamma-astro-data-formats.readthedocs.io/en/v0.3/general/coordinates.html#field-of-view

They say:

> FOV LON should increase with decreasing OTHER LON

Reply:
But that's true for both `FOV_ALIGNMENTS`, so `sky_to_fov` should already produce the correct sign

Reply:
sky_to_fov already has a `-` for fov_lon, so here should be None, which is the actual fix

Reply:
Would still be important that the coordinates come out the same way as they did before (which is still what we expect, right?)

Reply:
No, this was opened to make gammapy match the GADF definition, which wasn't the case before

Reply:
Does the HESS background models match the GADF definition ?
If not we need a special handling to still be able to use them.